build_tools/cmake/iree_check_test.cmake

```diff
@@ -79,18 +79,18 @@ function(iree_check_test)
     iree_modules_check_iree-check-module
   )
 
-  string(REPLACE "_" "/" _PACKAGE_PATH ${_PACKAGE_NAME})                                                                                                    
-  set(_NAME_PATH "${_PACKAGE_PATH}:${_RULE_NAME}") 
+  string(REPLACE "_" "/" _PACKAGE_PATH ${_PACKAGE_NAME})
```

Comment:
Did this change it to windows line endings or something? :nauseated_face:  I don't see a diff

Reply:
you had trailing whitespace

Reply:
Ah thanks :-/